pallets/bucket-nfts/Cargo.toml

```diff
@@ -30,6 +31,7 @@ sp-std = { workspace = true }
 
 # Local
 shp-traits = { workspace = true }
+shp-file-key-verifier = { workspace = true }
```

Comment:
If you only need this for the `mock.rs` file, please add them as `dev-dependencies`.